packages/core/core.ts

```diff
@@ -165,18 +164,43 @@ function logInfoAboutFuzzerOptions(fuzzerOptions: string[]) {
 	});
 }
 
+function makeSigintCallback(options: Options, sync: boolean) {
+	const stopNativeFuzzer = sync ? Fuzzer.stopFuzzing : Fuzzer.stopFuzzingAsync;
+	return () => {
+		stopFuzzing(
+			undefined,
+			options.expectedErrors,
+			options.coverageDirectory,
+			options.coverageReporters,
+			options.sync
+		);
+		// In case no error was thrown, we need to stop the native fuzzer manually.
+		stopNativeFuzzer(0);
```

Comment:
`stopFuzzing` should always calls `Fuzzer.stopFuzzing[Async]` internally.

Reply:
I wondered the same thing too, but the fuzzer does not exit without this! Looking at the code superficially, we only stop libfuzzer in case of an error on the JavaScript side, and simply return on no error?

Reply:
Ah, yes, you're right. If no error was raised and none was expected the process can exit normally after libFuzzer finishes. I added an optional parameter to enforce a shutdown with a dedicated exit code.